src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -172,6 +172,29 @@ const FacilityInputForm: React.FC<Props> = ({ scenarioId }) => {
     navigate("/");
   };
 
+  const includesKnownAges = (inputs: Record<string, any>) => {
+    let definedInputs = omitBy(inputs, isUndefined);
+    let ageKnownInputs = pickBy(definedInputs, (_, key) => {
+      return RegExp(/age\d+/).test(key);
+    });
+    let total = Object.values(ageKnownInputs).reduce((sum, n) => {
+      return sum + n;
+    }, 0);
+    return total > 0;
+  };
+
+  const includesUnknownAges = (inputs: Record<string, any>) => {
+    let definedInputs = omitBy(inputs, isUndefined);
+    let ageUnknownInputs = pickBy(definedInputs, (_, key) => {
+      return RegExp(/ageUnknown\w+/).test(key);
+    });
+    let array = Object.values(ageUnknownInputs);
+    let total = array.reduce((sum, n) => {
+      return sum + n;
+    }, 0);
+    return total > 0;
+  };
```

Comment:
since this is repeated more or less verbatim (along with the toast text) I think you should factor it out into a function that you can import into both of these files ... maybe it could throw an error that you could catch in order to interrupt the save